src/applications/hca/components/FormDescriptions/index.jsx

```diff
@@ -643,8 +643,20 @@ export const PolicyOrGroupDescription = (
   </div>
 );
 
+export const PolicyOrGroupTitle = (
+  <>
+    Insurance policy number or group code.&nbsp;
+    <span className="schemaform-required-span vads-u-font-family--sans vads-u-font-size--base vads-u-font-weight--normal">
+      (*Required)
+    </span>
+  </>
+);
+
 export const TricarePolicyDescription = (
-  <va-additional-info trigger="I have TRICARE. What’s my policy number?" uswds>
```

Comment:
The `uswds` attribute forced the components into V3 styling during the transition from V1. Since the default is V3, this attribute is no longer necessary.